### ISSUE TITLE
[SFS] Show only sectors form selected sector division

### ISSUE THREAD

**Issue by u-mueller** [OpenEnergyPlatform/oekg#33]
## Description of the issue

If you choose a predifined sector division, say KSG, then only KSG sectors should show up und field: "Which sectors are considered in the study?". This function has been before, but seems to have vanished.

## Steps to Reproduce
1. Choose predifined sector division
2. Check list of sectors
3.

## Ideas of solution

1) Only allow subclasses/individuals from predifined sector divisions
2) Only show the general sectors below, when non of the listed sector divisions is chosen (?):
- Agriculture, forestry and land use sector
- Energy demand sector
- Building sector
- Commercial sector
- Heating and cooling sector
- Household sector
- Transport sector
- Energy transformation sector
- Electricity sector
- Industry sector

## Context and Environment
* Version used: latest
* Operating system: Windows 10, Firefox 102.10.0esr, MS Edge Version 112.0.1722.48
* Environment setup and (python) version: 

## Workflow checklist
- [ ] I am aware of the workflow in [CONTRIBUTING.md](CONTRIBUTING.md)

**Comment by u-mueller** [comment]
Maybe this also relates to https://github.com/OpenEnergyPlatform/oekg/issues/32, maybe we structure ´sector division´ and ´sector´ equivalent to ´energy carrier´ and ´energy transformation process´.

**Comment by stap-m** [comment]
@adelmemariani has this functionality been deactivated by accident?

**Comment by adelmemariani** [comment]
Yes, this functionality has been deactivated since there was a bug while opening the factsheet for the second time. This means the sectors belonging to the selected sector divisions should be shown and marked (and expanded when there is a hierarchy). I just made all sectors available temporarily. I will try to activate this functionality for the next release.
